### ISSUE TITLE
VS code suporte.

### ISSUE THREAD

**Issue by emersonbottero** [open-wc/custom-elements-manifest#272]
**Checklist**
- [ x ] Did you run the analyzer with the `--dev` flag to get more information?
- [ ] Did you create a **minimal** reproduction in the [playground](https://custom-elements-manifest.netlify.app/)?

Completing the items above will greatly improve triaging time of your issue.

**Expected behavior**
Once setting UP settings.json in the .vscode file I would expect ir to show docs info while hovering an custom element.

I saw an example hello-world in the vscode-custom-data repository that works but the structure doesn't match:

Has field version:1.1 instead o schemaVersion: 1.0.0
Has field tags: instead of modules:

Maybe better documenting it or creating an example.

**Comment by break-stuff** [comment]
@emersonbottero are you using a plugin like this to generate the VS Code-formatted data?
https://www.npmjs.com/package/custom-element-vs-code-integration

**Comment by emersonbottero** [comment]
@break-stuff  I'm using this one
https://www.npmjs.com/package/@custom-elements-manifest/analyzer

**Comment by emersonbottero** [comment]
no.. now I wrote a script to translate the json create with this lib to the vscode

**Comment by break-stuff** [comment]
Nice! Is it similar to this plugin? Is it something I can view?

**Comment by break-stuff** [comment]
@emersonbottero you can also use the [Web Components Language Server](https://wc-toolkit.com/integrations/web-components-language-server/) to eliminate the translation and setup steps.
